src/app/notifications/components/SearchFailure/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { memo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { assertWindow, useOnDOMEvent, useTimeout } from '../../../utils';
+import { Header } from '../Card';
+import {
+  BannerBody,
+  BannerBodyWrapper,
+  clearErrorAfter,
+  CloseButton,
+  CloseIcon
+} from './styles';
+
+interface Props {
+  dismiss: () => void;
+}
+
+const window = assertWindow();
+
+// tslint:disable-next-line:variable-name
+const SearchFailure = memo(({ dismiss }: Props) => {
```

Comment:
why are you using memo here?

Reply:
I thought that since callback that triggers the render gets executed few times it might cause some issues with the timeout, my bad.

Reply:
no worries, i was wondering if i was missing something